genie_registry/vcf.py

```diff
@@ -18,6 +18,8 @@ class vcf(FileTypeFormat):
     _fileType = "vcf"
 
     _process_kwargs = []
+    _allele_col = "REF"
```

Comment:
We can just make it consistent and accept a list here.  That said, ALT is actually another allele column.  It's a little more complicated, so I think we should leave it out for now and see how genome nexus handles some of the files that aren't formatted properly

> ALT - alternate base(s): Comma separated list of alternate non-reference alleles. These alleles do not have to
be called in any of the samples. Options are base Strings made up of the bases A,C,G,T,N,*, (case insensitive)
or an angle-bracketed ID String (“<ID>”) or a breakend replacement string as described in the section on
breakends. The ‘*’ allele is reserved to indicate that the allele is missing due to a upstream deletion. If there
are no alternative alleles, then the missing value should be used. Tools processing VCF files are not required
to preserve case in the allele String, except for IDs, which are case sensitive. (String; no whitespace, commas,
or angle-brackets are permitted in the ID String itself)

Reply:
Yeah, I didn't think we would handle the alternative non-ref alleles. This looks like fun to handle if we did.